### PR TITLE
Exp/distortion loss

### PR DESCRIPTION
**Pull request by fedelopez77**
Adding two different Distortion losses to experiment with.

This loss complements the distance-based loss by minimizing the distortion between positive samples of user-item pairs. 
These are two alternatives, as proposed in https://arxiv.org/abs/1809.01703

### CODE REVIEW

**Comment by fedelopez77** [comment]
Done!

File: hyperbolic-rs/rudders/losses.py [hunk 1]
```diff
@@ -52,3 +50,26 @@ def calculate_loss(self, model, input_batch):
     def loss_from_distances(self, dist_to_pos, dist_to_neg):
         """distances are negative. This means d = -dist(x,y)"""
         return tf.reduce_mean(tf.nn.relu(self.margin - dist_to_pos + dist_to_neg))
+
+
+class HingeDistortionLoss(PairwiseHingeLoss):
+    """This loss can only be used with DistanceDistortionHyperbolic since the model need to implement
```
[the remark this answers is not in the document]
Comment (fedelopez77) [reply hyperbolic-rs/rudders/losses.py]: Done!
